portfolio/src/main/webapp/mapScript.js

```diff
@@ -15,6 +15,7 @@
 let map;
 
 function showMarkers() {
+  initMap(true);
```

Comment:
Is it possible to split this function into `initMap` and `initMapWithMarkers` (the latter could just call the former as a subroutine).
Because it's not obvious, what `true` stands for here.

Reply:
initMap is pretty big function, can i create some variable like markerFlag instead of copying it entirely?

Reply:
I think what Ivan is suggesting is not to copy initMap, but to remove the parameter `markerFlag`. There is only a small amount of code that depends on `markerFlag` which can be moved to a new function (`initMapWithMarkers`) . This function should first call `initMap` and then run the rest of the code to add the markers.

Where you currently call `initMap(true)` you can call `initMapWithMarkers`. Where you call `initMap(false)` you can then just call `initMap`.